WorkloadManagementSystem/DB/JobDB.py

```diff
@@ -277,8 +277,10 @@ def getJobParameters( self, jobID, paramList = None ):
         if not ret['OK']:
           return ret
         paramNameList.append( ret['Value'] )
-      paramNames = ','.join( paramNameList )
-      cmd = "SELECT Name, Value from JobParameters WHERE JobID=%s and Name in (%s)" % ( e_jobID, paramNames )
+      if len( paramNameList ) == 1:
```

Comment:
So I checked, you don't need a specific case for one element, the ```in``` clause is optimzied:
```
MySQL [FileCatalogDB]> explain select * from FC_StorageElements where SEName in ('SARA-USER');
+----+-------------+--------------------+-------+---------------+--------+---------+-------+------+-------+
| id | select_type | table              | type  | possible_keys | key    | key_len | ref   | rows | Extra |
+----+-------------+--------------------+-------+---------------+--------+---------+-------+------+-------+
|  1 | SIMPLE      | FC_StorageElements | const | SEName        | SEName | 129     | const |    1 | NULL  |
+----+-------------+--------------------+-------+---------------+--------+---------+-------+------+-------+
1 row in set (0.01 sec)

MySQL [FileCatalogDB]> explain select * from FC_StorageElements where SEName = 'SARA-USER';
+----+-------------+--------------------+-------+---------------+--------+---------+-------+------+-------+
| id | select_type | table              | type  | possible_keys | key    | key_len | ref   | rows | Extra |
+----+-------------+--------------------+-------+---------------+--------+---------+-------+------+-------+
|  1 | SIMPLE      | FC_StorageElements | const | SEName        | SEName | 129     | const |    1 | NULL  |
+----+-------------+--------------------+-------+---------------+--------+---------+-------+------+-------+
1 row in set (0.00 sec)

MySQL [FileCatalogDB]> explain select * from FC_StorageElements where SEName in ('SARA-USER', 'SARA-DST');
+----+-------------+--------------------+-------+---------------+--------+---------+------+------+-----------------------+
| id | select_type | table              | type  | possible_keys | key    | key_len | ref  | rows | Extra                 |
+----+-------------+--------------------+-------+---------------+--------+---------+------+------+-----------------------+
|  1 | SIMPLE      | FC_StorageElements | range | SEName        | SEName | 129     | NULL |    2 | Using index condition |
+----+-------------+--------------------+-------+---------------+--------+---------+------+------+-----------------------+
1 row in set (0.00 sec)
```